Classes/Service/Provider/GoogleMapsProvider.php

```diff
@@ -35,7 +35,7 @@ public function encodeAddress(array $parameter, array $settings): array
         }
 
         $apiUrl = $settings['geocodeUrl'] .
-            (!empty($settings['apiConsoleKey']) ? '&key=' . $settings['apiConsoleKey'] : '') .
+            (!empty($settings['apiConsoleKeyGeocoding']) ? '&key=' . $settings['apiConsoleKeyGeocoding'] : '') .
```

Comment:
I would prefere to have a fallback behaviour for atleast one major version with deprecation warning for those upgrading an existing installation.